### PR TITLE
Create 07_Regularization directory

### PR DESCRIPTION
**Pull request by gathurian**
Created Exercise-stuff

### CODE REVIEW

**Review by KilnOfTheSecondFlame — CHANGES_REQUESTED** [review]
Could you please rename the notebooks to follow our scheme?
00_Exercises Readme
01_Exercise 01
02_Exercise02

File: 07_Regularization/00_Exercises Readme.ipynb [hunk 1]
```diff
@@ -61,17 +47,19 @@
...
+   "source": [
+    "4. Print the corresponding coefficients of the regression polynomial and draw it."
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "4. Print the corresponding coefficients of the regression polynomial and draw it."
+    "# Exercise 4: Real World Examples\n",
```
Comment (KilnOfTheSecondFlame) [inline 07_Regularization/00_Exercises Readme.ipynb]: Interesting numbering 1,4, ... , ?

**Review by KilnOfTheSecondFlame — CHANGES_REQUESTED** [review]

File: 07_Regularization/00_Exercises Readme.ipynb (hunk 1 above)
Comment (gathurian) [reply 07_Regularization/00_Exercises Readme.ipynb]: [Really](https://tenor.com/view/bruh-gif-5156041)

**Review by KilnOfTheSecondFlame — APPROVED** [review]

File: 07_Regularization/00_Exercises Readme.ipynb [hunk 2]
```diff
@@ -57,17 +57,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Exercise 4: Real World Examples\n",
+    "# Exercise 2: Real World Examples\n",
```
Comment (KilnOfTheSecondFlame) [inline 07_Regularization/00_Exercises Readme.ipynb:60]: Good boy 😄

**Review by KilnOfTheSecondFlame — APPROVED** [review]

**Pull request merged**